src/core/entity.js

```diff
@@ -57,19 +52,44 @@ class Entity {
       37: 'r_l'
     };
 
-    math.vec3.subtract(this.forward, look_at, this.position);
-    math.vec3.cross(this.right, this.forward, up);
-    math.vec3.cross(this.up, this.right, this.forward);
-
-    math.vec3.normalize(this.forward, this.forward);
-    math.vec3.normalize(this.right, this.right);
-    math.vec3.normalize(this.up, this.up);
-
     if (this.vertices && this.indices && this.normals) {
       this.create_buffers();
     }
   }
 
+  look_at(vec, up = this.up) {
```

Comment:
Should we call this `target_to`?

Reply:
I don't think so - `look_at` is self explanatory, I believe in this case we're having it better than `gl-matrix`. Keep in mind that Unity's `look_at` is doing exactly what ours is doing.